### ISSUE TITLE
Teams integration

### ISSUE THREAD

**Issue by simonfelding** [robusta-dev/kubernetes-chatgpt-bot#1]
Your readme says to open an issue for teams integration. I'd love that!

**Comment by aantn** [comment]
No ETA yet, we're looking into it!

We need to make some improvements upstream to [Robusta itself](https://github.com/robusta-dev/robusta) to support interactive buttons in MSTeams.

**Comment by simonfelding** [comment]
Cool! You're aware of this, right? 

https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/universal-actions-for-adaptive-cards/up-to-date-views

Maybe I'll start work on a patch for this. If you're already working on one, could you maybe start a pull request?

**Comment by aantn** [comment]
Yep! We're already using the adaptive cards interface - https://github.com/robusta-dev/robusta/tree/master/src/robusta/integrations/msteams

The challenge is with receiving a callback from MSTeams when a button is pressed. We don't have an action-receiver mechanism setup yet for Teams.

**Comment by dsputnikk** [comment]
+1 for this. Would be amazing 👏

**Comment by mcerverc** [comment]
It would be nice Teams integration :)
